src/ex1/script.js

```diff
@@ -0,0 +1,129 @@
+const addTaskButton = document.querySelector(".add-task");
+const taskInput = document.querySelector("#task-text");
+const allTasksContainer = document.querySelector(".all-tasks-container");
+const pendingTasksText = document.querySelector(".pending-tasks");
+let pendingTasks = 0;
+const clearButton = document.querySelector(".clear-all");
+const sortButton = document.querySelector(".name-sort");
+
+// Event trigger when user clicks the add task button
```

Comment:
relevant to all comments: when you add a comment think about the person who first reads this code. does he really need this comment to understand what the code does?
- if the answer is yes - it probably means the code is not readable enough and it may can be refactored to make it more readable (you will learn about it later on in the course).
- if the answer is no - it means that the reader reads the code twice - once when he reads the comment, and right after that when he reads the actual code.

a good rule of thumb is to write comments only when you need to explain **why** you did that, and not **what** you did

Reply:
you split the code into functions and your code is very readable, in this case we don't need those comments 🙂

Reply:
Great. thanks, I was unaware of this:) will take them off